src/Bicep.Core.Samples/Files/baselines_bicepparam/Invalid_Parameters/parameters.diagnostics.bicepparam

```diff
@@ -6,7 +6,35 @@ param para1 = 'value
 //@[14:20) [BCP004 (Error)] The string at this location is not terminated due to an unexpected new line character. (CodeDescription: none) |'value|
 
 para
-//@[00:04) [BCP007 (Error)] This declaration type is not recognized. Specify a metadata, parameter, variable, resource, or output declaration. (CodeDescription: none) |para|
+//@[00:04) [BCP337 (Error)] This declaration type is not valid for a Bicep Parameters file. Specify a "using" or "param" declaration. (CodeDescription: none) |para|
 
-para2
-//@[00:05) [BCP007 (Error)] This declaration type is not recognized. Specify a metadata, parameter, variable, resource, or output declaration. (CodeDescription: none) |para2|
+para2
+//@[00:05) [BCP337 (Error)] This declaration type is not valid for a Bicep Parameters file. Specify a "using" or "param" declaration. (CodeDescription: none) |para2|
+
+param expr = 1 + 2
+//@[00:18) [BCP259 (Error)] The parameter "expr" is assigned in the params file without being declared in the Bicep file. (CodeDescription: none) |param expr = 1 + 2|
+//@[13:18) [BCP338 (Error)] Complex expressions are not permitted in a Bicep Parameters file. (CodeDescription: none) |1 + 2|
+
+param interp = 'abc${123}def'
+//@[00:29) [BCP259 (Error)] The parameter "interp" is assigned in the params file without being declared in the Bicep file. (CodeDescription: none) |param interp = 'abc${123}def'|
+//@[15:29) [BCP338 (Error)] Complex expressions are not permitted in a Bicep Parameters file. (CodeDescription: none) |'abc${123}def'|
+
+param doubleinterp = 'abc${interp + 'blah'}def'
+//@[00:47) [BCP259 (Error)] The parameter "doubleinterp" is assigned in the params file without being declared in the Bicep file. (CodeDescription: none) |param doubleinterp = 'abc${interp + 'blah'}def'|
+//@[21:47) [BCP338 (Error)] Complex expressions are not permitted in a Bicep Parameters file. (CodeDescription: none) |'abc${interp + 'blah'}def'|
+//@[27:33) [BCP063 (Error)] The name "interp" is not a parameter, variable, resource or module. (CodeDescription: none) |interp|
```

Comment:
Do we want show a bicepparam specific message (like "interp" is not a parameter) and remove words variable, resource or module?

Or not worry about it since the user would always see the complex expression in this case?

Reply:
This is a great catch - I've decided to just rework and go with the simpler fix to address it.